### ISSUE TITLE
event.dataTransfer is undefined

### ISSUE THREAD

**Issue by EpokK** [angular-dragdrop/angular-dragdrop#22]
Under the last Chrome version, the `dataTransfer` attribute is `undefined` on `event` object.

**Comment by Toilal** [comment]
Is this still an issue on current master ?

**Comment by jaskaye17** [comment]
Yes this is still a problem. I have implemented @larvata 's workaround everywhere that e.dataTransfer is present in the code

**Comment by ganarajpr** [comment]
@Toilal This should be fixed now ?

**Comment by Toilal** [comment]
I'm not sure it's fixed in the whole library. 

I've just merged an undefined check on `e.dataTransfer`, but maybe it's required in more places.

**Comment by Toilal** [comment]
But it seems to occur only on windows chrome, according to #130. So i can't really test.

**Comment by mortenson** [comment]
I get this on OS X Chrome. Specifically, I was able to replicate the issue by writing a `ui-on-Drop` callback like this:

```
$scope.onDrop = function($event, $data, previous_index) {
  $scope.foo.push($data);
  $scope.bar.splice(previous_index, 1);
}
```

Basically, remove something $data from its previous place in the scope and add it to a new place in the scope. I haven't tried to replicate this without the rest of my application, but can do so if needed.

**Comment by mortenson** [comment]
Here's my implementation of @larvata's fix: https://github.com/mortenson/angular-dragdrop/commit/69449b4aa8d4e67f47d6af12c1b2f158fb83ce73

**Comment by BuonOmo** [comment]
I’ve had the same issue, but not working with angular : only jquery. So maybe It has no link with angular ?

**Comment by cladicovr** [comment]
@BuonOmo  I am having the same issue. I use knockout.js.
